extensions/binding/org.eclipse.smarthome.binding.onewire/src/main/java/org/eclipse/smarthome/binding/onewire/internal/Util.java

```diff
@@ -71,6 +78,13 @@ public static State calculateAbsoluteHumidity(QuantityType<Temperature> temperat
         }
     }
 
+    /**
+     * calculates the dewpoint in °C from measured values
+     *
+     * @param temperature      the measured temperature
```

Comment:
```suggestion
     * @param temperature the measured temperature
```

Reply:
I accepted your suggestion. The formatting is due to #6424.